.gitignore

```diff
@@ -9,3 +9,4 @@
 /.tox/
 
 /docs/_build/
+__pycache__
```

Comment:
Sure no problem, though it's a little weird that a Python project wouldn't ignore __pycache__ directories by default.  GitHub provides a pretty good default .gitignore file [that I used for my Paperless project](https://github.com/danielquinn/paperless/blob/master/.gitignore) if you wanna have a look.  (Obviously, the latter part of the file is stuff I added that's Paperless-specific).